pom.xml

```diff
@@ -43,7 +43,7 @@
     <checkstyle.config.location>src/conf/checkstyle/configuration.xml</checkstyle.config.location>
     <checkstyle.suppressions.location>src/conf/checkstyle/suppressions.xml</checkstyle.suppressions.location>
     <jacoco.coverage.ratio>0.46</jacoco.coverage.ratio>
-    <jacoco.missed.count>55</jacoco.missed.count>
+    <jacoco.missed.count>65</jacoco.missed.count>
```

Comment:
The increase is due to extra generated flyweight classes for metadata request and response